lab-integration/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "env": {
+    "node": true,
+    "es6": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier"
+  ],
+  "plugins": ["@stylistic/js", "@typescript-eslint"],
```

Comment:
`plugins` is to add a rule pre-fined in a plugin. Since we don't use any rule from `@typescript-eslint`, can we remove it?

Also, can we bring back `root`? The doc says it's a good practice.

Ref: https://typescript-eslint.io/getting-started/legacy-eslint-setup#details